automol/vmat.py

```diff
@@ -66,36 +76,33 @@ def from_data(symbs: tuple[str, ...], key_mat, name_mat=None, one_indexed=None):
 
 # # V-Matrix/V-Matrix common functions (document these as z-matrix functions)
 # # # getters
-def symbols(vma, idxs: list[int] | None = None) -> list[str]:
+def symbols(vma: VMatrix, idxs: list[int] | None = None) -> list[str]:
     """Obtain the atomic symbols for all atoms defined in the V-Matrix.
 
     :param vma: V-Matrix
-    :type vma: automol V-Matrix data structure
     :param idxs: indices of atoms to obtain information for
-    :type idxs: List[int]
-    :returns: The list of atomic symbols
-    :rtype: List[str]
+    :return: The list of atomic symbols
     """
     if vma:
-        symbs, *_ = tuple(zip(*vma))
+        symbs, *_ = tuple(zip(*vma, strict=False))
     else:
         symbs = ()
 
     return symbs if idxs is None else tuple(map(symbs.__getitem__, idxs))
 
 
-def key_matrix(vma, shift=0):
+def key_matrix(vma: VMatrix, shift: int = 0) -> KeyMatrix:
     """Obtain the key matrix of the V-Matrix that contains the
     coordinate atom keys by row and column.
 
     :param vma: V-Matrix
     :type vma: automol V-Matrix data structure
```

Comment:
As we add type annotations, we can take out all of these `:type` and `:rtype:` fields in the doc string, since they are redundant and in many cases wrong.